pyairtable/orm/model.py

```diff
@@ -170,6 +172,29 @@ def _validate_class(cls) -> None:
                 )
             )
 
+    def modify_kwargs(func):
+        # Modifies kwargs passed to Model.all and Model.first
+        # If Meta.use_field_ids, kwargs['return_fields_by_field_id']=True.
+
+        @wraps(func)
+        def wrapper(cls, *args, **kwargs):
+            kwargs = cls._kwargs_all_first(kwargs)
+            return func(cls, *args, **kwargs)
+
+        return wrapper
```

Comment:
This shouldn't be inside the Model class – though I'm not sure it's really necessary. The `all` and `first` methods can do something like this and it's only one line of code:

```python
def all(self, **kwargs) -> ...:
    kwargs.update(self._get_meta_request_kwargs())
    ...
```